apps/web/hooks/escrow/use-Escrow.ts

```
@@ -0,0 +1,131 @@
+}
+
+export const useEscrow = (escrowId: string) => {
+  const [isLoading, setIsLoading] = useState(false)
+  const [error, setError] = useState<string | null>(null)
+  const [escrowData, setEscrowData] = useState<EscrowRecord | null>(null)
+
+  const handleError = (err: unknown): string => {
+    return err instanceof Error ? err.message : 'An error occurred'
+  }
+
+  const fetchEscrowStatus = useCallback(async () => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: handleError

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)